packages/web-components/src/components/promo-group/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,217 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+import'../promo-group';
+
+# `<dds-promo-group>`
```

Comment:
Just to stay consistent with the other docs, let's have the titles be `Promo Group` and `Promo Item` and not the element name